test-packages/platform-feature-tests/cypress/support/commands.js

```diff
@@ -99,3 +109,16 @@ Cypress.Commands.add('uploadFileFromFixtureDirectory', (filename) => {
 	})
 	cy.get('input[type="file"]').selectFile(`cypress/fixtures/${filename}`, { force: true });
 });
+
+
+Cypress.Commands.add('checkIfUnchecked',(labelText) => {
+	cy.contains('label',labelText).invoke('attr','for').then((id) => {
+		cy.get(`#${id}`).then($checkbox =>{
+			if(!$checkbox.is(':checked')){
+				cy.wrap($checkbox).click();}
```

Comment:
A minor point, but the formatting is a little confusing here in terms of line breaks and where the curly braces sit

Reply:
Indentation applied